### DISCUSSION TITLE
Migrating to another server

### DISCUSSION THREAD

**Discussion by ZiYueCommentary** [WeblateOrg/weblate#12357] [Q&A] [answered]
I am migrating my Weblate to another server. I am using docker compose and I successfully migrated the database by following [this doc](https://docs.weblate.org/en/weblate-5.7/admin/install/docker.html#upgrading-postgresql-container). But weblate internal git repo could not find any translation file, even though translations are still in the website. Seems I need to migrate `weblate-data` too, but simply copy-and-paste is not working. I didn't find a migration tutorial for this. What should I do? 🙏

**Comment by nijel** [comment, marked as answer]
Yes, the data content needs to be migrated as well. The documentation you've referenced covers PostgreSQL upgrades only. Backup and migration is covered here: https://docs.weblate.org/en/weblate-5.7/admin/backup.html

**Reply by ZiYueCommentary** [reply]
I successfully migrated it. Thank you!
